nl_server/lib/nl_embeddings.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Managing the embeddings."""
+from datasets import load_dataset
+from google.cloud import storage
+from sentence_transformers import SentenceTransformer
+from sentence_transformers.util import semantic_search
+from typing import Dict, List, Union
+
+import logging
+import os
+import torch
+
+GCS_BUCKET = 'datcom-nl-models'
+EMBEDDINGS = 'prod/'
+EMBEDDINGS_FILENAME = "embeddings_us_filtered_2023_1_26_22_15_41.csv"
```

Comment:
@shifucun once we start looking for the latest timestamp, we can get rid of EMBEDDGINS_FILENAME constant

Reply:
sg!